### ISSUE TITLE
Warning: a promise was created in a handler at ... but was not returned from it

### ISSUE THREAD

**Issue by chrisabrams** [mbroadst/rethunk#1]
I get this error about a hundred times when starting an app using rethunk:

```
(node:7) Warning: a promise was created in a handler at app/node_modules/rethunk/lib/cursor.js:134:10 but was not returned from it, see http://goo.gl/rRqMUw
    at new Promise (/app/node_modules/bluebird/js/release/promise.js:77:14)
```

Is this an app development issue, or is this a library issue?

**Comment by mbroadst** [comment]
@chrisabrams its a warning from bluebird, you could try adding a `return p;` after the creation of the promise, or adding `return null;` after `self._push(response);` @ `rethunk/lib/cursor.js:134`

**Comment by mbroadst** [comment]
fixed in cc45f11
